code/src/index.css

```diff
@@ -5,9 +5,188 @@ body {
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+.playlist-album-wrapper {
+  display: flex;
+}
+.playlists-wrapper {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  border-right: 1px solid rgba(73, 71, 71, 0.911);
+  padding: 5px;
+  width: 100px;
+}
+.image-playlists {
+  width: 80%;
+  position: relative;
+}
+.playlist-h2 {
+  font-size: 12px;
+  margin-bottom: 20px;
+  text-align: left;
+  color: #fff;
+}
+.playlist-link {
+  display: flex;
+  flex-direction: column;
+  text-decoration: none;
+  color: #ffffff;
+  margin: 5px 0px 0px 0px;
+  font-size: 12px;
+  font-family: Helvetica;
+  margin: 0 0 10px 0;
+  width: 80%;
+  text-align: left;
+}
+.playlist-link:hover {
+  text-decoration: underline;
+}
+.playlist-image {
+  width: 100%;
+}
+
+.header h1 {
+  font-family: Helvetica, sans-serif;
+  font-size: 19px;
+  color: #fff;
+}
+.header {
+  border-bottom: 2px solid rgba(85, 84, 84, 0.623);
+  margin: auto auto 10px 90px;
+}
+.album-name {
+  font-family: Helvetica, sans-serif;
+  color: #fff;
+}
+p {
+  font-size: 14px;
+  margin: 5px 0 0 0;
+}
+.album-name:hover {
+  text-decoration: underline;
+}
+.artist-names a {
+  text-decoration: none;
+  font-family: Helvetica, sans-serif;
+  color: #a0a0a0;
+  display: inline-block;
+}
+.artist-names a:hover {
+  text-decoration: underline;
+}
+.artist-name p {
+  font-size: 14px;
+  padding: 0;
+}
+
+.list {
+  color: #fff;
+}
+.artist-box {
+  display: flex;
+  flex-direction: row;
+}
+.album-box {
+  width: 260px;
+  height: 100%;
+  margin: 10px;
+}
+.article-box {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+.album-wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-items: center;
+}
+.artist-names:not(:first-child, :last-child)::before {
+  color: #a0a0a0;
+  content: ", ";
+}
+.artist-names:last-child::before {
+  content: " & ";
+  color: #a0a0a0;
+}
+.artist-names:only-child::before {
+  content: "";
+}
+.album-img {
+  width: 100%;
+  height: 100%;
+}
+.icon-image-wrapper {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+.icon-wrapper {
+  justify-content: space-around;
+  align-items: center;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  position: absolute;
+  background-color: rgba(0, 0, 0, 0.644);
+  display: none;
+}
+.icon-image-wrapper:hover .icon-wrapper {
+  display: flex;
+}
+
+.icon {
+  width: 30px;
+  filter: invert(100%);
+}
+.play-icon {
+  width: 70px;
+  opacity: 0.6;
+}
+
+.play-icon:hover {
+  opacity: 1;
+  width: 75px;
+  margin: -2.5px;
+  overflow: hidden;
+  transition: ease-in-out 0.2s;
+}
+
+@media (min-width: 768px) {
+  .playlists-wrapper {
+    width: 150px;
+  }
+  .header {
+    margin-left: 140px;
+  }
+  .album-wrapper {
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    align-items: baseline;
```

Comment:
Never used align-items: baseline, does it do smth specific to the layout? Will also google about it 👍🏼